src/NHibernate/Logging.extensions.cs

```diff
@@ -20,7 +20,7 @@ public static void Fatal(this INHibernateLogger logger, Exception exception, str
 
 		public static void Fatal(this INHibernateLogger logger, Exception exception, string message)
 		{
-			logger.Log(NHibernateLogLevel.Fatal, new NHibernateLogValues(message, null), exception);
+			logger.Log(NHibernateLogLevel.Fatal, new NHibernateLogValues(message, Array.Empty<object>()), exception);
```

Comment:
Couldn't this be handled in the struct constructor, which is already doing this for its `message` argument?

Reply:
Good point.  Changed and rebased.